### ISSUE TITLE
Articles consisting of a teaser only have a "Read more" link

### ISSUE THREAD

**Issue by Anke** [contao/contao#4872]
### Affected version(s)

4.13.5

### Description

Hi,

articles with teasers only are automatically providing a Read more link. When I click on the Read more link I see a pagfe with Return link.

Example: In the demo I added another article and defined both to display their teasers. While the demo article contains content, the added article only consists of the teaser and shouldn't have a Read on link. Thus I'd say the `<?php if ($this->teaserOnly): ?>` isn't working (properly).

![2022-06-24_154832_contao](https://user-images.githubusercontent.com/1300920/175549420-d4e3faf2-dcb4-472b-aecf-1b8e4e299941.jpg)

By the way - I think it would be useful to be able to select a headline level for the teaser in the backend.

**Comment by aschempp** [comment]
Why would you add a teaser and enable the "show teaser" checkbox option if you don't want a readmore link? You could simply add that content to the article content instead?

**Comment by Anke** [comment]
Okay, then I'll have to teach my folks to not use article teasers for very short articles. I guess I misinterpreted the line `<?php if ($this->teaserOnly): ?>`? 

There are some sports clubs among my customers who are also using news teasers for short one-line-infos, where there's really no need to have a link to a separate page repeating this same short line of text, not even for SEO. Thus we always changed 
`<?php if ($this->hasText || $this->hasTeaser): ?>`
to
`<?php if ($this->hasText): ?>`
Years ago I opted for a possibility to detemine whether a user wants a "Read more" link or not, but although I know we're not the only ones handling news teasers this way, regretfully our suggestion didn't find developers' approval.
